src/bin/server/app.rs

```diff
@@ -43,14 +42,16 @@ mod test {
     use crate::attestations::{
         AggregatePriceIntervalEntry, OracleMessage, PriceIntervalEntry, PriceValueEntry,
     };
+    use crate::db::get_db_url;
     use axum::{body::Body, http::Request};
     use bls::{AggregateSignature, SecretKey, Signature};
     use bytes::Bytes;
+    use itertools::Itertools;
     use tower::ServiceExt;
 
-    async fn get_db_pool() -> SqlitePool {
-        let test_db_url = "sqlite::memory:";
-        let pool = SqlitePool::connect(test_db_url).await.unwrap();
+    async fn get_db_pool() -> DbPool {
```

Comment:
Removed it. Good point regarding warnings, we might want to extend our ci to fail on warnings, so that we are forced to either fix them or acknowledge them using one of the `allow` macros